src/Makefile.xtables

```diff
@@ -6,7 +6,11 @@ clean:
 	rm -f $(LIBS)
 
 install:
-	install -g root -o root -m 644 $(LIBS) /lib/xtables/
+	if [ -d /usr/lib/x86_64-linux-gnu/xtables ]; then \
+      install -g root -o root -m 644 $(LIBS) /usr/lib/x86_64-linux-gnu/xtables; \
+    else \
+      install -g root -o root -m 644 $(LIBS) /lib/xtables/ ; \
+    fi
```

Comment:
please change it so that the default directory is /lib/xtables/ not  /usr/lib/x86_64-linux-gnu/xtables.
also fix indentation please.